pykechain/client.py

```diff
@@ -1378,6 +1378,60 @@ def create_proxy_model(
         )
         return self._create_part(action='create_proxy_model', data=data, **kwargs)
 
+    def _create_parts_bulk(
+            self,
+            parts: List[Dict],
+            **kwargs
+    ) -> PartSet:
+        """
+        Create multiple part instances simultaneously.
+
+        :param parts: list of dicts, each specifying a part instance. Available fields per dict:
+            * name: str
+            * parent_id: str of parent instance
+            * model_id: str of part model
+            * properties: list of dicts, each specifying a property to update. Available fields per dict:
+                * name: str
+                * value: Any
+                * model_id: str of property model
+        :param kwargs:
+        :return: list of Part instances
+        :rtype list
+        """
+        check_list_of_dicts(
```

Comment:
does this raise a `ValueError` or `IllegalArgumentError` in case it is not right?

- [ ] Yes it does
- [ ] No it does not and I need to add some `raise`-ing somewhere.

Reply:
raises IllegalArgumentError, and my tests are checking for that